lib/modem_trace/modem_trace.c

```diff
@@ -90,13 +93,15 @@ int modem_trace_start(enum modem_trace_mode trace_mode, uint16_t duration, uint3
 		k_timer_start(&trace_stop_timer, K_SECONDS(duration), K_SECONDS(0));
 	}
 
+	is_trace_session_in_progress = true;
+
 	return 0;
 }
 
 int modem_trace_process(const uint8_t *data, uint32_t len)
 {
-	if (!is_transport_initialized) {
-		return -ENXIO;
+	if (!is_trace_session_in_progress) {
```

Comment:
This change will block the possibility of reading out modem traces during boot.

Reply:
yeah i overlooked that. I will fix this.